report.ipynb

```diff
@@ -123,6 +123,7 @@
    "outputs": [],
    "source": [
     "INSTRUMENT_VENDOR = os.getenv(\"INSTRUMENT_VENDOR\", \"Illumina\")\n",
+    "ARTIC_PRIMER_VERSION = os.getenv(\"ARTIC_PRIMER_VERSION\", \"4\")\n",
```

Comment:
I would default to "unknown" here if `ARTIC_PRIMER_VERSION` is not defined.

Reply:
We need to assume a primer version to do alignment trimming and the default is v4. We could modify the report to say "assuming v4 primers"?

Reply:
Will change to `os.environ["ARTIC_PRIMER_VERSION"]` so it blows up if  `ARTIC_PRIMER_VERSION` is undefined